### DISCUSSION TITLE
Autoscalingrunnerset deleted during v0.9.3 upgrade

### DISCUSSION THREAD

**Discussion by jfoechsler** [community/community#160704] [Actions]
Hello. Wondering how it could happen that Autoscalingrunnersets were deleted by Helm upgrade.
I did an upgrade retry with a dummy value change and objects were created again.

Regards

**Comment by rekha-prakash-maersk** [comment]
I tried to upgrade from 0.9.2 to 0.9.3 and only controller instances came up with new version but listener pod were removed and not started again with new version. I had to uninstall all the helm release to clean up and reinstalled them. I am using flux + kustomize
